online/src/app/store/state/auth.state.ts

```diff
@@ -0,0 +1,13 @@
+import { User } from "src/app/user";
+
+export interface AuthentificationState {
+    loggedUser: User | null;
+    error: string | null;
+    status: 'pending' | 'loading' | 'error' | 'success';
```

Comment:
You can also create a type for this status. But oftentimes it's just represented as a simple loading boolean.